src/extension.ts

```diff
@@ -34,15 +34,24 @@ import { existsSync, rmSync } from 'fs';
 import { CodemodConfig } from './data/codemodConfigSchema';
 import { parsePrivateCodemodsEnvelope } from './data/privateCodemodsEnvelopeSchema';
 import { SourceControlPanelProvider } from './components/webview/SourceControlPanelProvider';
-
-export const UrlParamKeys = {
-	engine: 'engine' as const,
-	beforeSnippet: 'beforeSnippet' as const,
-	afterSnippet: 'afterSnippet' as const,
-	codemodSource: 'codemodSource' as const,
-	codemodName: 'codemodName' as const,
-	codemodHashDigest: 'chd' as const,
-};
+import {
+	AlreadyLinkedError,
+	GlobalStateTokenStorage,
+	UserService,
+} from './components/userService';
+
+export const enum SEARCH_PARAMS_KEYS {
+	ENGINE = 'engine',
+	BEFORE_SNIPPET = 'beforeSnippet',
+	AFTER_SNIPPET = 'afterSnippet',
+	CODEMOD_SOURCE = 'codemodSource',
+	CODEMOD_NAME = 'codemodName',
+	COMMAND = 'command',
+	COMPRESSED_SHAREABLE_CODEMOD = 'c',
+	CODEMOD_HASH_DIGEST = 'chd',
+	ACCESS_TOKEN_REQUESTED = 'accessTokenRequested',
```

Comment:
is that needed, are you using `command` now?

Reply:
addressed